.githooks/post-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+git cliff --bump > CHANGELOG.md
```

Comment:
correct behaviour here will probably be dependent on how git-cliff is configured on each users machine, right?. Could we add a git-cliff configuration in the repo?

Reply:
there is a cliff.toml in here, was added in https://github.com/famedly/hookd/commit/9550ed699cd2d85f2c150c52757fd3d2210d1425 and then modified in https://github.com/famedly/hookd/commit/f1a6bcb9dbaea8765e84de34ff185bdb2a6b96bf